test_launcher.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
I have one of those "old" systems where python refers to python2.7 and  python3 is used to run python-3 programs.

One way to fix is this is let CMake find the python3 interpreter path and have it insert it here using configure_file

Reply:
Fixed